README.md

```diff
@@ -171,6 +171,12 @@ Run the `destroy-cloudfoundry` pipeline.
 
 # Additional notes
 
+## Running tests locally
+
+You will need to install some dependencies to run the unit tests on your own
+machine. The most up-to-date reference for these is the Travis CI
+configuration in [`.travis.yml`](.travis.yml).
+
```

Comment:
Alex and I agreed to leave this as-is for now. We can add more detail later, at the cost of duplication, if people are having problems.